taworks/taform/templates/taform/application.html

```diff
@@ -1,30 +1,140 @@
 {% load static %}
 <script src="{% static 'taform/js/app.js' %}" type="text/javascript" ></script>
-<link rel="stylesheet" type="text/css" href="{% static 'taform/css/style.css' %}" />
 
+<link rel="stylesheet" type="text/css" href="{% static 'taform/css/style.css' %}" />
 
 <body onload="noBack();" onpageshow="if (event.persisted) noBack();" onunload="">
 {% load taform_extras %}
+
+<h1>Management Sciences TA Application Form</h1>
+
 <form enctype="multipart/form-data" autocomplete="off" action="." method="POST" accept-charset="utf-8" >
+
 {% csrf_token %}
-<table>
-{{ s_form }}
+<font color = "red">{{error}}</font>
```

Comment:
remove color = "red",

I believe Laura's branch has the css that using font tag will give red. double check that, or else add this to css styling.